spec.html

```diff
@@ -24492,7 +24492,7 @@ <h1>The Object Constructor</h1>
         <li>is the initial value of the *"Object"* property of the global object.</li>
         <li>creates a new ordinary object when called as a constructor.</li>
         <li>performs a type conversion when called as a function rather than as a constructor.</li>
-        <li>is designed to be subclassable. It may be used as the value of an `extends` clause of a class definition.</li>
+        <li>may be used as the value of an `extends` clause of a class definition.</li>
```

Comment:
this is definitely better! but:
```suggestion
        <li>may be used as the value of an `extends` clause of a class definition, but is not intended to be subclassed.</li>
```
perhaps? (at least for all the non-nullish primitive constructors, where it just doesn't make sense to ever subclass them)